SECURITY.md

```diff
@@ -0,0 +1,42 @@
+# Security Policy
+
+## Supported Versions
+
+`authd` is currently released to a PPA. There are two PPAs, [stable](https://launchpad.net/~ubuntu-enterprise-desktop/+archive/ubuntu/authd) and [edge](https://launchpad.net/~ubuntu-enterprise-desktop/+archive/ubuntu/authd-edge)
+We provide security updates for the following versions of `authd`. Please ensure you are using a supported version to receive updates and patches.
+
+If you are unsure of your version, please run the following command in a terminal and check which PPA `authd` is installed from:
+
+```shell
+apt policy authd
+```
+
+## Reporting a Vulnerability
+
+If you discover a security vulnerability within this repository, we encourage responsible disclosure. Please report any security issues to help us keep `authd` secure for everyone.
+
+### Private Vulnerability Reporting
+
+The most straightforward way to report a security vulnerability is via [GitHub](https://github.com/canonical/lxd/security/advisories/new).
```

Comment:
This is the lxd repo, we know where you copied from! :p

Reply:
ah,  I've read it ten times :D

Reply:
I’m unsure the 11th times would have fixed it TBH :p Always better with fresh eyes!